subprojects/platform-base/src/main/java/org/gradle/language/base/internal/tasks/StaleOutputCleaner.java

```diff
@@ -79,4 +79,31 @@ public static boolean cleanOutputs(Deleter deleter, Iterable<File> filesToDelete
 
         return outputsCleaner.getDidWork();
     }
+
+    @CheckReturnValue
+    public static boolean cleanEmptyOutputDirectories(Deleter deleter, Iterable<File> filesToDelete, File directoryToClean) {
+        return cleanEmptyOutputDirectories(deleter, filesToDelete, ImmutableSet.of(directoryToClean));
+    }
+
+    @CheckReturnValue
+    public static boolean cleanEmptyOutputDirectories(Deleter deleter, Iterable<File> directories, ImmutableSet<File> directoriesToClean) {
```

Comment:
Both variant of this method should have the same name for this parameter.

```suggestion
    public static boolean cleanEmptyOutputDirectories(Deleter deleter, Iterable<File> filesToDelete, ImmutableSet<File> directoriesToClean) {
```

Reply:
fixed